layer/layer_data.h

```diff
@@ -353,6 +365,10 @@ class LayerData {
       VkDevice device, const VkShaderModuleCreateInfo* create_info,
       const VkAllocationCallbacks* allocator, VkShaderModule* shader_module);
 
+  void DestroyShaderModule(
```

Comment:
Could you add documentation for this function? Something brief like by the functions above.